velox/docs/functions/map.rst

```diff
@@ -26,6 +26,12 @@ Map Functions
 
     See also :func:`map_agg` for creating a map as an aggregation.
 
+.. function:: map_from_entries(array(row(K, V))) -> map(K, V)
```

Comment:
Functions in the documentation should appear in alphabetical order. Hence, map_from_entries should come after map_entries. Please, update.

Reply:
> Functions in the documentation should appear in alphabetical order. Hence, map_from_entries should come after map_entries.

I think the function order in the documentation follows the [presto doc order](https://prestodb.io/docs/current/functions/map.html?highlight=map_zip_with#map_from_entries) where `map_from_entries` is ahead of `map_entries`. It doesn't follow alphabetical order. An anti example is: `map_entries` -> `map_concat` -> `map_filter`.

Reply:
This looks like a bug in Presto's doc.

Reply:
I'm not sure it makes sense to introduce this function in the wrong place. Velox and Presto are separate projects. We don't need to duplicate Presto bugs :-)

Reply:
Ok, I'm happy to adjust the function order in map.rst.

Reply:
Thank you.